### ISSUE TITLE
Graph list view - No Graphs Found

### ISSUE THREAD

**Issue by eschoeller** [Cacti/cacti#450]
To reproduce this issue:
- Select graphs tab
- Select list view tab
- Select a single device from the Device drop-down menu
- Select any number of available graphs
- Hit view
[Now the message "No Graphs Found" is displayed, and no graphs]
- Hit "Go"
[Now all those graphs you had selected show up!]

I don't think you should need to hit "Go", the graphs should just pop up the first time instead of saying "No Graphs Found". 

I have been using Firefox and Chrome, both have the same issue.

**Comment by cigamit** [comment]
Resolved, thanks for reporting!

**Comment by eschoeller** [comment]
This problem is back, but it's worse this time. I can't apply any filters in the list view and get them to display. I can only manually select graphs without filters and they'll view OK

**Comment by eschoeller** [comment]
This issue is back...

**Comment by cigamit** [comment]
Need repeatable case.

**Comment by eschoeller** [comment]
- Select graphs tab
- Select list view tab
- Select a single template (or multiple) from the drop down.
- Hit the "x" to close the drop down
[all graphs for that template now load]
- Select a few of these graphs
- Hit the "View" button

"No Graphs Found" message is displayed. Still reproducible if I select just a single device at this point as well.

**Comment by eschoeller** [comment]
I just went through my original procedure, and that still works, so I suppose this is likely a very similar, but different "No Graphs Found" issue. Sorry perhaps I should have opened a new issue.

**Comment by eschoeller** [comment]
Sorry I've been off on other things for awhile. I just upgraded to 1.1.16 and was hoping this issue was resolved ... but I'm still having the same problem. One work-around I found is that when I'm presented with the graph view page and the "No Graphs Found" message, the Template field will say "1 Template Selected". If I now remove this and say "All" it changes to "523 Templates Selected" and I now am able to see the graphs I had originally selected on the previous screen. 
For the past couple months I've been creating dummy graph trees and using the graph management function to put the graphs I want to view onto the dummy tree so I can view them. Having the list view function work again would be super great.
